internal/postgres/config.go

```diff
@@ -100,6 +103,12 @@ func Environment(cluster *v1beta1.PostgresCluster) []corev1.EnvVar {
 			Name:  "PGPORT",
 			Value: fmt.Sprint(*cluster.Spec.Port),
 		},
+		// Setting the KRB5_CONFIG for kerberos
+		// - https://web.mit.edu/kerberos/krb5-devel/doc/admin/conf_files/krb5_conf.html
```

Comment:
💯 I love links. ⛏️ Can we link to the "current" docs instead?

```suggestion
		// - https://web.mit.edu/kerberos/krb5-current/doc/admin/conf_files/krb5_conf.html
```